### PR TITLE
Feat/remove private org

### LINKED ISSUE #1604: API - Remove Support for Private Organizations
**Is your feature request related to a problem? Please describe.**
We need to reduce and simplify some of the Talawa features. 

The concept of private organizations is unlikely given that most organizations using Talawa will be non-profits where transparency will be important.

**Describe the solution you'd like**

Remove all support for private organizations in:
1. Backend systems and supporting code

Other:

1. All tests must be valid and pass
1. No other functionality must be affected.

**Describe alternatives you've considered**
- N/A

**Approach to be followed (optional)**
- See above

**Additional context**
Related issues include:
1. https://github.com/PalisadoesFoundation/talawa-api/issues/1604
2. https://github.com/PalisadoesFoundation/talawa-admin/issues/1327
3. https://github.com/PalisadoesFoundation/talawa/issues/2287

**Potential internship candidates**
- Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by AVtheking**
**What kind of change does this PR introduce?**
Removes support for private organizations

**Issue Number:**

Fixes #1604 

**Did you add tests for your changes?**
NO

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
Merge your code with the latest upstream.

None of the files in the `.gitHub` directory must be included in this PR. It is out of scope

**Comment by palisadoes** [comment]
The `INSTALLATION.md` file must not be included in this PR. It is out of scope

**Comment by xoldyckk** [comment]
@palisadoes is a public organization an organization that is publicly available for searching, but it cannot be joined without approval of the admins?

File: src/models/Organization.ts [hunk 1]
```diff
@@ -125,9 +119,9 @@ const organizationSchema = new Schema({
       ref: "User",
     },
   ],
-  visibleInSearch: {
-    type: Boolean,
-  },
+  // visibleInSearch: {
```
Comment (xoldyckk) [inline src/models/Organization.ts:122]: remove the code instead of just commenting it, same goes all the instances in this PR

Comment (AVtheking) [reply src/models/Organization.ts]: I have removed the code

**Comment by AVtheking** [comment]
I have created another pr for this please review that

**Comment by palisadoes** [comment]
No. This PR must remain.

We have a history here and we now need to find reviewers again. 

It is also against the correct branch.

Please be considerate of our volunteers time.

There is no reason to choose this at all

**Comment by palisadoes** [comment]
I'm reopening and closing whichever one you reopen.

This is in such bad form

**Comment by AVtheking** [comment]
Sorry sir for the inconveniences but because of the huge no of conflicts I have to delete and re upload the branch, that's why I have raised another pr

**Comment by palisadoes** [comment]
> Sorry sir for the inconveniences but because of the huge no of conflicts I have to delete and re upload the branch, that's why I have raised another pr

You could still do it with the existing PR and branch.

I'll reopen it soon

**Comment by AVtheking** [comment]
Ok sir

**Comment by Cioppolo14** [comment]
@AVtheking We cannot reopen the PR. Please open a new one, and then do not close it.  Do not close a PR in the future unless your reviewers explicitly request that. This has created a lot of extra work for our team.

**Pull request closed without merging**